tools/SourceKit/tools/sourcekitd/lib/API/Requests.cpp

```diff
@@ -3181,6 +3181,22 @@ class CompileTrackingConsumer final : public trace::TraceConsumer {
 };
 } // end anonymous namespace
 
+static UIdent getUIDForOperationKind(trace::OperationKind OpKind) {
+  static UIdent CompileOperationPerformSema("source.compile.operation.perform-sema");
```

Comment:
Maybe we should just leave off the kind when it's from the ASTManager? This name is not specific (every AST build except code-completion uses performSema), and it is meaningless to  a user.

Reply:
Done